docs/DeveloperGuide.md

```diff
@@ -195,6 +210,36 @@ The following sequence diagram shows how a list is autosorted upon an addition o
     * Pros: Relatively low time complexity i.e. O(logn).
     * Cons: Prone to error and difficult to implement.
 
+### Done feature
+
+#### Implementation
+
+This Done feature allows users to mark their deliverables as completed. The user input is received by`MainWindow` in
+the `UI` component before being passed to `DeliverableLogicManager` to be executed. The `DeliverableLogicManager` will call
```

Comment:
the UI component

Reply:
I think here it refers to the big UI class so shld have ``